packaging/nix/flake.nix

```diff
@@ -2,53 +2,38 @@
   description = "umu universal game launcher";
 
   inputs = {
-    nixpkgs = {
-      type = "github";
-      owner = "NixOS";
-      repo = "nixpkgs";
-      ref = "nixpkgs-unstable";
-    };
+    # FIXME:
+    # upstream package is in nixpkgs-unstable & nixos-unstable-small
+    # it will be in nixos-unstable soon
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
   };
 
-  outputs = {
-    self,
-    nixpkgs,
-  }: let
-    umu-launcher-src = builtins.toPath "${self}/../../";
-
-    pkgs = import nixpkgs {
-      system = "x86_64-linux";
-      overlays = [self.overlays.default];
-    };
-
-    version = "1.1.4";
-  in {
-    overlays.default = final: prev: let
-      py = prev.python3;
-    in {
-      umu-launcher = final.callPackage ./umu-launcher.nix {
-        inherit version;
-        umu-launcher = umu-launcher-src;
-        pyth1 = py;
+  outputs =
+    { self, nixpkgs }:
+    let
+      pkgs = import nixpkgs {
+        system = "x86_64-linux";
+        overlays = [ self.overlays.default ];
```

Comment:
We could totally just avoid importing nixpkgs and overlays altogether. This flake exposes one package, please.

Reply:
The flake has always exposed an overlay, so removing it is a breaking change.

Also, IDK how you can create a package that depends on nixpkgs without instantiating nixpkgs. Perhaps I've misinterpreted?

Reply:
Easily: `pkgs = inputs.nixpkgs.legacyPackages.${system}`

Reply:
That is still instantiating an instance of nixpkgs

Reply:
Sure, I'm in favor of using a nixpkgs instance that is likely to have already been evaluated.

I should be able to refactor things such that both the overlay and the package outputs use some common package implementation, without the package output having to depend on the overlay.

Reply:
Didn't fufe add the overlay recently? I'm still in favor of just removing it